piv/key.go

```diff
@@ -210,6 +214,9 @@ func parseAttestation(slotCert *x509.Certificate) (*Attestation, error) {
 			return nil, fmt.Errorf("parsing extension: %v", err)
 		}
 	}
+	subjectSplit := strings.Split(slotCert.Subject.CommonName, " ")
```

Comment:
Why the split? Is this documented anywhere? Can you link to that documentation in a comment? :)

Also, please add a bounds check:

```
if len(subjectSplit) > 0 {
  a.Slot = subjectSplit[len(subjectSplit)]
}
```